pkg/services/grpcserver/service.go

```diff
@@ -47,15 +48,28 @@ func ProvideService(cfg *setting.Cfg, features featuremgmt.FeatureToggles, authe
 		enabled: features.IsEnabledGlobally(featuremgmt.FlagGrpcServer),
 	}
 
+	histogramOptions := prometheus.HistogramOpts{
+		Namespace: "grafana",
+		Name:      "grpc_request_duration_seconds",
+		Help:      "Time (in seconds) spent serving HTTP requests.",
+		Buckets:   instrument.DefBuckets,
+	}
+
+	if features.IsEnabledGlobally(featuremgmt.FlagEnableNativeHTTPHistogram) {
```

Comment:
This is a gRPC histogram, not an HTTP histogram. I think you should use a new feature toggle for this, or maybe even none (this is low-impact, and compatible with clients that don't understand the new exposition format)